docs/architecture/cloud-native/definition.md

```diff
@@ -168,77 +168,63 @@ Beyond the guidance provided in the 12-factor manifesto, there are several criti
 
 To take full advantage of the underlying cloud infrastructure and it’s built features and services, we need a way to monitor and invoke the special features.
 
-*Cloud-based Monitoring*
-
-Blah
-
-*Cloud-based Scaling*
-
-Blah
-
-*Cloud-based Resiliency*
-
-Blah
-
-*Self-Healing*
-
-Blah
+We discuss monitoring, scaling, resiliency, and self-healing in this book
 
 ### Orchestration
 
-If you have a handful of microservices, your DevOps might be able to manually manage the workload. They would need to pay close attention to the performance and traffic patterns, so they could manually scale accordingly. If, on the other hand, you have 10, 100 or even 1,000 services, all bets are off.
+If you had only a handful of microservices, your DevOps *might* be able to manually manage the workload. They would need to pay close attention to the performance and traffic patterns, so they could manually scale accordingly. If, on the other hand, you have 10, 100 or even 1,000 services, all bets are off.
 
-When you’re operating at scale, [container orchestration](https://blog.newrelic.com/engineering/container-orchestration-explained/), i.e., automating the deployment, management, scaling, networking, and availability of your containers – is mandatory.
+When you’re operating at scale, [container orchestration](https://blog.newrelic.com/engineering/container-orchestration-explained/) is mandatory. Orchestration includes deployment, management, scaling, networking, and availability of your containers.
 
-Container orchestration is done with a special software program called (you guessed it) a container orchestrator. While many are available, [Kubernetes](https://kubernetes.io/) has become the de facto standard in the cloud native world. The Azure cloud features Kubernetes as a managed service, [Azure Kubernetes Service](https://azure.microsoft.com/services/kubernetes-service/), which means that you are not responsible for provisioning or maintaining the Kubernetes infrastructure – a career path in and of itself.
+Container orchestration is done with a special software program called (you guessed it) a container orchestrator. While several are available, [Kubernetes](https://kubernetes.io/) has become the de facto standard for the cloud native world. The Azure cloud features Kubernetes as a managed service, [Azure Kubernetes Service](https://azure.microsoft.com/services/kubernetes-service/), which means that Kubernetes is provided to you as a service (Platform as a Service). You are not responsible for provisioning or maintaining the Kubernetes infrastructure – a career path in and of itself.
 
-What exactly do containers orchestrators do? Figure 1-10, shown below, provides some insight.
+What exactly do containers orchestrators do? Figure 1-10 provides some insight.
 
 ![What container orchestrators do](media/what-container-orchestrators-do.png)
 **Figure 1-10**. What container orchestrators do
```

Comment:
line 236, you have a broken button because of bad formatting 
It should be > not \<

Reply:
Resolved